src/test/resources/application-integrationtest.properties

```diff
@@ -14,7 +14,10 @@ it.targetApplicationName=integrationapp
 it.sourceStorageAccountName=tdrintegrationsrc1
 it.ingestRequestContainer=ingestrequests/test
 
-#users
+# QA Common Test Users
+# If you need to log in as one of these users (ex. if integration tests are failing due to a need to
+# accept Terms of Service), the password can be found in Vault:
```

Comment:
@snf2ye I know you said that you had to use this password recently.  I'm curious what you needed to do? I'd be happy to add another example.

Reply:
When you run TDR UI E2E tests locally, you need to auth as the test user (see the CYPRESS_GOOGLE_TOKEN doc here - https://github.com/DataBiosphere/jade-data-repo-ui?tab=readme-ov-file#prerequisites). So, it would probably be good to add this line there as well!